packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -167,29 +160,17 @@ async function convertFHIRBundleToMedicalRecord({
     dateTo,
     conversionType,
   };
-  const inactiveLambdaPayload: ConversionInput = {
-    ...activeLambdaPayload,
-    resultFileNameSuffix: inactiveSuffix,
-  };
 
   const [result] = await Promise.all([
     lambdaClient
       .invoke({
-        FunctionName: activeLambdaName,
+        FunctionName: lambdaName,
         InvocationType: "RequestResponse",
         Payload: JSON.stringify(activeLambdaPayload),
       })
       .promise(),
-    inactiveLambdaName &&
-      lambdaClient
-        .invoke({
-          FunctionName: inactiveLambdaName,
-          InvocationType: "RequestResponse",
-          Payload: JSON.stringify(inactiveLambdaPayload),
-        })
-        .promise(),
```

Comment:
this is the removal of the actual invocation, right? whats the historical reason we left this in?

Reply:
We were running both in parallel to compare results and be able to migrate cxs progressively.